crates/ruff/src/importer.rs

```diff
@@ -180,7 +186,7 @@ fn end_of_statement_insertion(stmt: &Stmt, locator: &Locator, stylist: &Stylist)
         // Otherwise, insert on the next line.
         Insertion::new(
             "",
-            locator.line_end(location),
+            locator.full_line_end(location),
```

Comment:
whoops  :hand_over_mouth: ... my bad. Thanks for fixing this and adding tests.

Reply:
Happy to help! :)